api/tests/app/db.py

```diff
@@ -663,3 +676,24 @@ def ses_notification_callback():
            'dd426d95ee9390147a5624348ee.pem",' \
            '\n  "UnsubscribeURL" : "https://sns.eu-west-1.amazonaws.com/?Action=Unsubscribe&S' \
            'subscriptionArn=arn:aws:sns:eu-west-1:302763885840:preview-emails:d6aad3ef-83d6-4cf3-a470-54e2e75916da"\n}'
+
+
+def create_invited_user(service=None,
+                        to_email_address=None):
+
+    if service is None:
+        service = create_service()
+    if to_email_address is None:
+        to_email_address = 'invited_user@digital.gov.uk'
```

Comment:
gov.au

Reply:
might leave this as is for now as there's a few places in the codebase we can change this over to au, but also it may not matter.